### DISCUSSION TITLE
any way to turn off holiday theme???

### DISCUSSION THREAD

**Discussion by zahachtah** [fonsp/Pluto.jl#1794] [Questions about using Pluto] [answered]
Sorry, I appreciate the idea, but it gets old after a while.....

**Comment by zahachtah** [comment, marked as answer]
nvm :-)

**Comment by icweaver** [comment]
Totally just razzing you here, but your follow-up comment just reminded me of this:

![7khgt29lp1121](https://user-images.githubusercontent.com/25312320/147837609-88e268e8-e41c-424f-abf1-55b48f322e0c.jpg)
https://www.reddit.com/r/ProgrammerHumor/comments/a14wif/nvm_i_figured_it_out/

In all seriousness though for the other folks reading, there's a few things you can do:

1) Wait for the next release of Pluto
2) Use the current version (`pkg> add Pluto#main`)
3) Just press backspace on the current cell to reveal and delete the hidden html string

Anyway, hope you have a happy new year!
